sys/src/9/amd64/devacpi.c

```diff
@@ -1910,12 +1936,14 @@ static void acpiinitonce(void)
 	if (root != nil)
 		print("ACPI initialized\n");
 	addarchfile("acpimem", 0444, acpimemread, nil);
+	/* XXX these permissions are too loose? */
+	addarchfile("acpiio", 0666, acpiioread, acpiiowrite);
```

Comment:
Once we really turn security up, these can be 660 and only usable by eve. But it will do for now. Good to leave the note in.